stock_product_ul/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============
+Logistics Units
+===============
+
+The object *Logistics Units* (technical name *product.ul*) was defined in the official *product* module until Odoo v8 (cf `this code <https://github.com/odoo/odoo/blob/8.0/addons/product/product.py#L222>`_), but it was dropped in Odoo v9. This module restores this object and ensure compatibility for databases upgraded from Odoo v8 (same object name, same field names). This object allows to define the type of packages and/or pallets that you use to ship the goods. It may be needed to be able to display detailed information on packing lists and/or compute an accurate theorical weight per package that takes into account the weight of the packaging.
+
+It also restores the many2one field *Logistics Unit* (technical name *ul_id*) from *Packages* (technical name *stock.quant.packages*) to *Logistics Units* (technical name *product.ul*).
+
+If you use logistics units, you may also be interested by the module *stock_packaging_usability_ul* that allows to select the logistics unit on the fly when you define a new pack.
+
+Configuration
+=============
+
+Go to the Settings page of the Inventory menu and enable the option
```

Comment:
Better enumerated list for procedure

Reply:
Sure. But there is only one configuration to make here.

Reply:
see above (nice to have)